Pear/Profile/ProfileSectionTableViewCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ProfileSectionTableViewCell.swift
+//  Pear
+//
+//  Created by Lucy Xu on 3/13/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+
+class ProfileSectionTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private let titleLabel = UILabel()
+    private let itemsCollectionView = SelfSizingCollectionView()
+
+    static let reuseIdentifier = "ProfileSectionTableViewCell"
+    private var sectionType: ProfileSectionType? = nil
+    private var items: [String] = []
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        selectionStyle = .none
+        backgroundColor = .clear
+
+        titleLabel.font = .boldSystemFont(ofSize: 12)
+        titleLabel.textColor = .greenGray
+        contentView.addSubview(titleLabel)
+
+        itemsCollectionView.frame = CGRect(x: 0, y: 0, width: frame.width, height: 0)
```

Comment:
I haven't used a SelfSizingCollectionView() before, but do you need to give it a frame instead of doing .zero if you're going to constrain it later?